### DISCUSSION TITLE
Website UX is awful.

### DISCUSSION THREAD

**Discussion by beliboba** [community/community#140126] [GitHub Education] [answered]
### Hi everyone,

When trying to fill the application for github education, cant change cameras for taking the pictures. 
List of reasons of not being at campus is hilarious. Why cant my school just not have a campus? Should i live there or what?
Why do i even have to provide any reason of being far from it.

Also my application is getting rejected this year even though last 2 years github had been accepting them with no issues

**Comment by elhaddajiOtmane** [comment]
Try changing your browser and waiting a little bit. I had the same problem in Brave, but it works in Chrome

**Comment by ThisIs-Developer** [comment, marked as answer]
hey @beliboba,

sorry to hear about your experience! for the camera issue, try a different browser. if the campus requirements are frustrating, consider reaching out to GitHub support for clarification. as for the rejections, it’s worth contacting them to understand the change.

your feedback matters, so share it with GitHub too!

hope this helps!

---
if it does, mark it as the answer! 😪
